release notes/v0.50.0.md

```diff
@@ -38,7 +38,12 @@ _Format as `<number> <present_verb> <object>. <credit>`_:
 
 ## Maintenance and internal improvements
 
-- [browser#1164](https://github.com/grafana/xk6-browser/pull/1164), [browser#1171](https://github.com/grafana/xk6-browser/pull/1171), [browser#1173](https://github.com/grafana/xk6-browser/pull/1173), [browser#1175](https://github.com/grafana/xk6-browser/pull/1175), [browser#1179](https://github.com/grafana/xk6-browser/pull/1179), [browser#1183](https://github.com/grafana/xk6-browser/pull/1183), [browser#1186](https://github.com/grafana/xk6-browser/pull/1186), [browser#1186](https://github.com/grafana/xk6-browser/pull/1186), [browser#1188](https://github.com/grafana/xk6-browser/pull/1188), [browser#1189](https://github.com/grafana/xk6-browser/pull/1189), [browser#1191](https://github.com/grafana/xk6-browser/pull/1191), [browser#1193](https://github.com/grafana/xk6-browser/pull/1193), [browser#1161](https://github.com/grafana/xk6-browser/pull/1166), [browser#1190](https://github.com/grafana/xk6-browser/pull/1190), [browser#1163](https://github.com/grafana/xk6-browser/pull/1163) refactor internals to improve stability.
+- [browser#1164](https://github.com/grafana/xk6-browser/pull/1164), [browser#1171](https://github.com/grafana/xk6-browser/pull/1171), [browser#1173](https://github.com/grafana/xk6-browser/pull/1173),
+  [browser#1175](https://github.com/grafana/xk6-browser/pull/1175), [browser#1179](https://github.com/grafana/xk6-browser/pull/1179), [browser#1183](https://github.com/grafana/xk6-browser/pull/1183),
+  [browser#1186](https://github.com/grafana/xk6-browser/pull/1186), [browser#1186](https://github.com/grafana/xk6-browser/pull/1186), [browser#1188](https://github.com/grafana/xk6-browser/pull/1188),
+  [browser#1189](https://github.com/grafana/xk6-browser/pull/1189), [browser#1191](https://github.com/grafana/xk6-browser/pull/1191), [browser#1193](https://github.com/grafana/xk6-browser/pull/1193),
+  [browser#1161](https://github.com/grafana/xk6-browser/pull/1166), [browser#1190](https://github.com/grafana/xk6-browser/pull/1190), [browser#1163](https://github.com/grafana/xk6-browser/pull/1163),
```

Comment:
```suggestion
  [browser#1161](https://github.com/grafana/xk6-browser/pull/1161), [browser#1190](https://github.com/grafana/xk6-browser/pull/1190), [browser#1163](https://github.com/grafana/xk6-browser/pull/1163),
```

Reply:
Honestly, I'm not sure if it should be either one or the other 🙏🏻

Reply:
Nice catch! ⚾

Reply:
It should be #1166.

Reply:
```suggestion
  [browser#1166](https://github.com/grafana/xk6-browser/pull/1166), [browser#1190](https://github.com/grafana/xk6-browser/pull/1190), [browser#1163](https://github.com/grafana/xk6-browser/pull/1163),
```